code/game/objects/structures/artstuff.dm

```diff
@@ -354,3 +354,43 @@
 	for(var/x in 1 to width)
 		for(var/y in 1 to height)
 			grid[x][y] = I.GetPixel(w-x,h-y)
+
+//Presets for art gallery mapping, for paintings to be shared across stations
+/obj/structure/sign/painting/library1
+	persistence_id = "library1"
```

Comment:
Absolutely no reason to split it into that many categories. Just library, library_secure and library_private is more than enough.

Reply:
Are there no side effects from having multiple frames with the same persistence_id

Reply:
No, it's the intended usage.

Reply:
Neat!